modules/charon-core/src/main/java/org/wso2/charon3/core/attributes/SimpleAttribute.java

```diff
@@ -125,12 +122,12 @@ public Boolean getBooleanValue() throws CharonException {
 
     /*
      * uodate the attribute value
-     * 
+     *
      * @param value
-     * 
+     *
      * @throws CharonException
      */
-    public void updateValue(Object value) throws CharonException {
+    public void updateValue(Object value) {
```

Comment:
We are not recommending to change public methods as we need to preserve the backward compatibility as these public methods can be used from different other places.